content/o11y/openshift-logging-lokistack/index.md

```diff
@@ -0,0 +1,572 @@
+---
+date: '2021-07-27'
+title: Configuring OpenShift Logging using LokiStack on ROSA and (soon) ARO
+tags: ["Observability", "OCP"]
+authors:
+  - Andy Repton
+---
+
+A guide to shipping logs and metrics on OpenShift using the new LokiStack setup
```

Comment:
Is the purpose of this obvious to users? If not, you might add a bit more here to explain why someone would/should do this.